lib/Importer/ONNX.cpp

```diff
@@ -63,6 +63,17 @@ bool ONNXModelLoader::loadProtoFile(onnx::GraphProto &net,
   onnx::ModelProto MP;
   bool parseNet = MP.ParseFromCodedStream(&codedstr);
   net = MP.graph();
+  ir_version = MP.ir_version();
+  opset_version = 0;
+  if (ir_version >= 3) {
+    // Model with IR version < 3 doesn't have opset_import specified.
+    for (const auto &imp : MP.opset_import()) {
+      if (imp.has_domain() || imp.domain() == "") {
```

Comment:
What does this if check? Please add a comment on what are we trying to skip.

Did you mean `!` before first operand here? Because `imp.has_domain()` covers all cases: when `imp.domain() == ""` and also when it's not equal to ""